lib/jellyfish_web/telemetry/metrics_aggregator.ex

```diff
@@ -0,0 +1,116 @@
+defmodule JellyfishWeb.Telemetry.MetricsAggregator do
+  @moduledoc false
+
+  use GenServer
+
+  # in seconds
+  @metric_forwarding_interval 10
+
+  @ice_received_event [Membrane.ICE, :ice, :payload, :received]
+  @ice_sent_event [Membrane.ICE, :ice, :payload, :sent]
+
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
+  def start_link(opts),
+    do: GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+
+  @spec handle_event(term(), map(), map(), map()) :: :ok
+  def handle_event(name, measurements, metadata, config),
+    do: GenServer.cast(config.handler, {:event, name, measurements, metadata})
+
+  @spec metrics() :: [Telemetry.Metrics.t()]
+  def metrics() do
+    import Telemetry.Metrics
+
+    [
+      sum("jellyfish.traffic.ingress.total.bytes",
+        event_name: [:jellyfish],
+        measurement: :traffic_ingress_total
+      ),
+      last_value("jellyfish.traffic.ingress.throughput.bytes_per_second",
+        event_name: [:jellyfish],
+        measurement: :traffic_ingress_throughput
+      ),
+      sum("jellyfish.traffic.egress.total.bytes",
+        event_name: [:jellyfish],
+        measurement: :traffic_egress_total
+      ),
+      last_value("jellyfish.traffic.egress.throughput.bytes_per_second",
+        event_name: [:jellyfish],
+        measurement: :traffic_egress_throughput
+      ),
+      last_value("jellyfish.rooms"),
+      last_value("jellyfish.room.peers",
+        tags: [:room_id]
+      ),
+      sum("jellyfish.room.peer_time.total.seconds",
+        event_name: [:jellyfish, :room],
+        measurement: :peer_time_total,
+        tags: [:room_id]
+      )
```

Comment:
That might turn out to be problematic in the future as prometheus warns about high cardinality sets of labels https://prometheus.io/docs/practices/naming/#labels

Reply:
Yeah, for now I'll add a comment warning about this.

If I get any info back from Cloud on this issue before merging this, I'll post it here